### DISCUSSION TITLE
Image naming

### DISCUSSION THREAD

**Discussion by TzOk83** [erichelgeson/BlueSCSI-v1#24] [Q&A] [answered]
Hello!
Thank you for this great device, but one thing is unclear to me - how should I call the files if I want to have 2 images mounted:
HD10_512.hda and HD20_512.hda
or
HD10_512.hda and HD11_512.hda

**Comment by erichelgeson** [comment, marked as answer]
To keep it simple just go with the first - that will have a file served as a drive on SCSI id 1 and SCSI id 2.
